SimpleDrive.App/Extensions/WebHostExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using SimpleDrive.DAL;
+using SimpleDrive.DAL.Models;
+
+namespace SimpleDrive.App.Extensions
+{
+    public static class WebHostExtensions
+    {
+        public static void SeedData(this IWebHost host)
+        {
+            var scopeFactory = host.Services.GetRequiredService<IServiceScopeFactory>();
+            using (var scope = scopeFactory.CreateScope())
+            {
+                var db = scope.ServiceProvider.GetService<ApplicationDbContext>();
+
+                var roleManager = scope.ServiceProvider.GetRequiredService<RoleManager<Role>>();
+                var adminRole = roleManager.FindByNameAsync("Administrator").Result;
```

Comment:
make constants or move to config: `"Administrator"`, `"User`

Reply:
@nazar554 Fixed in 54f7112.